Formula/gam.rb

```diff
@@ -0,0 +1,16 @@
+class Gam < Formula
+  desc "Command-line management for Google G Suite"
+  homepage "https://github.com/jay0lee/GAM/wiki"
+  url "https://github.com/jay0lee/GAM/archive/v4.30.tar.gz"
+  sha256 "846b2902fe36019a9f4ebd4a911e780e103fb2d210971b8a4e407ddb6d8db4f6"
+  head "https://github.com/jay0lee/GAM.git"
+
+  def install
+    prefix.install Dir["*"]
```

Comment:
This should use libexec

Reply:
@ilovefs this has been updated to use libexec